### ISSUE TITLE
False positives?

### ISSUE THREAD

**Issue by rei-vilo** [wilhelmer/mkdocs-unused-files#1]
The **mkdocs-unused-files** throws many false positives.

The project contains different parts, called volumes, and each volume contains a sub-folder `img` for the pictures.

```
Project/
└── docs
    ├── img
    ├── Volume1
    │   └── img
    └── Volume2
        └── img
```

The plug-in is configured with

```
plugins:
    - unused_files:
        file_types:
        - png
        - jpg
        - svg
        - gif
        - jpeg
```

`Volume1/Chapter1.md` calls `![](img/Cat1.jpeg)` but the plug-in complains about `Volume1/img/Cat1.jpeg`.

```
INFO     -  The following files exist in the docs directory, but may be unused:
              - Volume1/img/Cat1.jpeg
```

Please find attached the minimal MkDocs project I am using to test **mkdocs-unused-files**. 

[Minimal_test_for_mkdocs-unused-files.zip](https://github.com/wilhelmer/mkdocs-unused-files/files/10236510/Minimal_test_for_mkdocs-unused-files.zip)

**Comment by wilhelmer** [comment]
First of all, thanks for testing, and thanks for the cats.

Second, as of now, `use_directory_urls` has to be set to `false` in `mkdocs.yml` for this plugin to work properly. I totally forgot to mention that, sorry. Also, I'm planning to release a version that will work regardless of the `use_directory_urls` setting.

So please try again with:

```yml
use_directory_urls: false
plugins:
    - unused_files:
        file_types:
        - png
        - jpg
        - svg
        - gif
        - jpeg
```

**Comment by rei-vilo** [comment]
Thank you for your prompt answer.

With new parameter `use_directory_urls: false`, output is

```
INFO     -  The following files exist in the docs directory, but may be unused:
              - img/Cat4.jpeg
```

Not correct: 

* None of the pictures are used by `index.md`

```
              - Volume1/img/Cat4.jpeg
              - Volume1/img/Cat3.jpeg
              - Volume1/img/Cat1.jpeg
              - Volume1/img/Cat2.jpeg
```

Not correct: 

* Cat1 and Cat3 are used by Volume1/Chapter1

```
              - Volume2/img/Cat4.jpeg
              - Volume2/img/Cat3.jpeg
              - Volume2/img/Cat1.jpeg
              - Volume2/img/Cat2.jpeg
```

Not correct: 

* Cat1 is used by Volume2/Chapter1
* Cat2 is used by Volume2/Chapter2

It looks like the plugin doesn't manage (yet) the local `img` sub-folders.

**Comment by wilhelmer** [comment]
I've released version 0.1.1 that should improve handling more complex directory structures like yours. Please check if that helps.

Also, it may now work with `use_directory_urls` set to `true`, but that's still experimental.

**Comment by rei-vilo** [comment]
With version 0.1.1 and whatever setting for  `use_directory_urls`, the results are consistent with the expectations. 

```
INFO     -  The following files exist in the docs directory, but may be unused:
              - img/Cat4.jpeg
              - img/Cat3.jpeg
              - img/Cat1.jpeg
              - img/Cat2.jpeg
```

Correct

```
              - Volume1/img/Cat4.jpeg
              - Volume1/img/Cat2.jpeg
```

Correct

```
              - Volume2/img/Cat4.jpeg
              - Volume2/img/Cat3.jpeg
```

Correct

I am keeping the cat-based example for later iterations.
